src/app/drive/services/download.service/downloadFile.ts

```diff
@@ -125,7 +105,16 @@ export default async function downloadFile(
     support = DownloadSupport.PatchedStreamApi;
   }
 
-  trackFileDownloadStart(userEmail, fileId, itemData.name, itemData.size, itemData.type, itemData.folderId);
+  const trackingDownloadProperties: TrackingPlan.DownloadProperties = {
+    file_download_id: analyticsService.getTrackingActionId(),
+    file_id: typeof fileId === 'string' ? parseInt(fileId) : fileId,
```

Comment:
Parsing a string with the hex format (like the fileId from the network) is going to give you 'NaN'